include/sensors/core/tasks/tip_presence_notification_task.hpp

```diff
@@ -29,7 +30,13 @@ class TipPresenceNotificationHandler {
             can::messages::PushTipPresenceNotification{
                 .message_index = 0,
                 .ejector_flag_status =
-                    static_cast<uint8_t>(hardware.check_tip_presence())});
+                    static_cast<uint8_t>(hardware.check_tip_presence()),
+                .sensor_id = sensor_id,
+            });
+        can_client.send_can_message(can::ids::NodeId::host,
```

Comment:
what's this new one for?

Reply:
that was old code I missed, removing that